manifests/init.pp

```diff
@@ -240,7 +244,24 @@
       default => '--no-sshd',
     }
 
-    require easy_ipa::validate_params
-    contain easy_ipa::install
+    if $adjust_login_defs {
+      file_line {
+        default:
+          path    => '/etc/login.defs',
+          replace => true,
+          ;
+        'adjust uid max':
+          line  => "UID_MAX\t11999",
+          match => '^UID_MAX\s*60000$',
```

Comment:
Perhaps the match should be more flexible, e.g. **^UID_MAX\s*\d*$**. That way if the value is not exactly 60000 file_line would not fail or worse, add additional lines because no match is was found.

Reply:
Also, does this apply to both FreeIPA clients and servers?